server/index.js

```diff
@@ -1,14 +1,33 @@
 const express = require('express');
+const httpProxy = require('http-proxy');
 
+const apiProxy = httpProxy.createProxyServer();
 const app = express();
 const port = 3010;
 
+// app.set('trust proxy', true);
 app.use(express.static('./public'));
 
 // const randNum = () => Math.floor(Math.random() * Math.floor(100));
 
-app.get('/', (req, res) => {
-  res.status(200).send('This is the proxy server!');
+app.get('/description', (req, res) => {
+  apiProxy.web(req, res, { target: 'http://localhost:3210' });
+});
+
+app.get('/totalReviews', (req, res) => {
+  apiProxy.web(req, res, { target: 'http://localhost:3004' });
+});
+
+app.get('/totalReviews', (req, res) => {
+  apiProxy.web(req, res, { target: 'http://localhost:3004' });
+});
+
+app.get('/allHomes', (req, res) => {
```

Comment:
oops... I should take this out of my module. I never ended up using it. Sorry!

Reply:
Thanks! I'll remove this too!